apps/ui/src/views/Space/Editor.vue

```diff
@@ -228,10 +239,10 @@ async function handleProposeClick() {
         choices,
         proposal.value.labels,
         appName.length <= 128 ? appName : '',
-        currentTime,
-        start,
-        minEnd,
-        maxEnd,
+        unixTimestamp.value,
+        proposalStart.value,
+        proposalMinEnd.value,
+        proposalMinEnd.value,
```

Comment:
This code is only valid for offchain spaces, for onchain spaces it's confusing.
1. We are passing minEnd for both minEnd and maxEnd - it will be correct for offchain spaces, but onchain spaces have two separate values.
2. Those values are timestamps, but EVM spaces actually use blocks instead. It was an issue even before this PR (due to recent change), so maybe for now just add comment here saying those are timestamps, not compatible with EVM spaces so it's less confusing.

Reply:
Added an additional variable for maxEnd. 

It was not introduced here as it's not doing anything, I was planning to introduce it only on the later PR (https://github.com/snapshot-labs/sx-monorepo/pull/1015/files#diff-4bc2c89f5ab53dbc871203c921f72d681cbf2673740878845e6269b26c77254cR204-R213)